### PR TITLE
Add battery fields for expert filters

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

File: src/translations/fr.json [hunk 1]
```diff
@@ -280,6 +280,8 @@
   "vlId": "ID poste",
   "p0": "P constant",
   "q0": "Q constant",
+  "batteryTargetP": "Puissance active constante",
+  "batteryTargetQ": "Puissance réactive constante",
```
Comment (thangqp) [inline src/translations/fr.json]: US are updated recently by Chistine, we do not need these keys, can reuse targetP, targetQ

In addition, we need another key, ie. 'filterNominalVoltage: Tension nominale poste' which is different to the existing key nominalVoltage, see https://github.com/gridsuite/gridexplore-app/pull/335/files#diff-d8294b74858af7afa897c6480b0ac7b32cdef156237e63c16bdbcea422f35b53R114

File: src/translations/en.json [hunk 2]
```diff
@@ -265,6 +265,8 @@
   "vlId": "Voltage level ID",
   "p0": "Constant P",
   "q0": "Constant Q",
+  "batteryTargetP": "Constant active power",
```
Comment (thangqp) [inline src/translations/en.json]: US are updated recently by Chistine, we do not need these keys, can reuse targetP, targetQ

In addition, we need another key, ie. 'filterNominalVoltage: Voltage level nominal voltage' which is different to the existing key nominalVoltage, see https://github.com/gridsuite/gridexplore-app/pull/335/files#diff-d8294b74858af7afa897c6480b0ac7b32cdef156237e63c16bdbcea422f35b53R114

File: src/components/dialogs/filter/expert/expert-filter-constants.ts [hunk 3]
```diff
@@ -247,4 +251,16 @@ export const fields: Record<string, Field[]> = {
...
+    BATTERY: [
+        FIELDS_OPTIONS.ID,
+        FIELDS_OPTIONS.NAME,
+        FIELDS_OPTIONS.VOLTAGE_LEVEL_ID,
+        FIELDS_OPTIONS.NOMINAL_VOLTAGE,
+        FIELDS_OPTIONS.CONNECTED,
+        FIELDS_OPTIONS.COUNTRY,
+        FIELDS_OPTIONS.MIN_P,
+        FIELDS_OPTIONS.MAX_P,
+        { ...FIELDS_OPTIONS.TARGET_P, label: 'batteryTargetP' },
```
Comment (thangqp) [inline src/components/dialogs/filter/expert/expert-filter-constants.ts]: No longer need batteryTargetP and batteryTargetQ, use direct targetP and targetQ

File: src/translations/fr.json (hunk 1 above)
Comment (AAJELLAL) [reply src/translations/fr.json]: Done.

File: src/translations/en.json (hunk 2 above)
Comment (AAJELLAL) [reply src/translations/en.json]: Done.

File: src/components/dialogs/filter/expert/expert-filter-constants.ts (hunk 3 above)
Comment (AAJELLAL) [reply src/components/dialogs/filter/expert/expert-filter-constants.ts]: FIxed.

**Review by thangqp — APPROVED** [review]
Code OK
Test OK

**Review by YenguiSeddik — APPROVED** [review]

**Pull request merged**